packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -580,11 +582,22 @@ export default class OverTheBoxDetailsCtrl {
     if (!this.serviceInfos || !this.serviceInfos.renew) {
       return false;
     }
-    return (
+    const canDelete =
       !this.serviceInfos.renew.deleteAtExpiration &&
       !this.serviceInfos.undoDeleteAtExpiration &&
-      this.serviceInfos.canDeleteAtExpiration
-    );
+      this.serviceInfos.canDeleteAtExpiration;
+    if (canDelete) {
+      this.atInternet.trackClick({
+        name: 'overTheBoxes::overTheBox::delete',
+        type: 'action',
+      });
+      this.atInternet.trackPage({
+        name: 'overTheBoxes::overTheBox::delete',
+        type: 'navigation',
+      });
```

Comment:
This hits should not be done into a method which test if the action button is displayed. (On `resiliate` or `$onInit` maybe ?)